cobc/typeck.c

```diff
@@ -2017,6 +2020,31 @@ cb_build_assignment_name (struct cb_file *cfile, cb_tree name)
 	}
 }
 
+cb_tree
```

Comment:
Please add a comment what this actually should build the name to giving one "special" example (and test case).